.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+name: Push docker images to Dockerhub
+
+on:
+  push:
+    branches: master
+    tags:
+      - "v*.*.*"
+
+jobs:
+  multi:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Build and push latestPattern Atlas Api
+        if: ${{ steps.vars.outputs.tag == 'master' }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: planqk/latex-renderer:latest
```

Comment:
instead of planqk/latex-renderer set the name as in docker hub, such as patternatlas/pattern-atlas-api:latest